grafana-plugin/src/components/Alerts/Alerts.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { Alert } from '@grafana/ui';
+import cn from 'classnames/bind';
+import { sanitize } from 'dompurify';
+
+import PluginLink from 'components/PluginLink/PluginLink';
+import { getSlackMessage } from 'containers/DefaultPageLayout/DefaultPageLayout.helpers';
+import { SlackError } from 'containers/DefaultPageLayout/DefaultPageLayout.types';
+import { getIfChatOpsConnected } from 'containers/DefaultPageLayout/helper';
+import { isTopNavbar } from 'plugin/GrafanaPluginRootPage.helpers';
+import { AppFeature } from 'state/features';
+import { useStore } from 'state/useStore';
+import LocationHelper from 'utils/LocationHelper';
+import { isUserActionAllowed, UserActions } from 'utils/authorization';
+import { GRAFANA_LICENSE_OSS } from 'utils/consts';
+import { useForceUpdate, useQueryParams } from 'utils/hooks';
+import { getItem, setItem } from 'utils/localStorage';
+
+import styles from './Alerts.module.scss';
+
+import plugin from '../../../package.json'; // eslint-disable-line
+
+const cx = cn.bind(styles);
+
+enum AlertID {
+  CONNECTIVITY_WARNING = 'Connectivity Warning',
+}
+
+export default function Alerts() {
+  const queryParams = useQueryParams();
+  const [showSlackInstallAlert, setShowSlackInstallAlert] = useState<SlackError | undefined>();
+
+  const forceUpdate = useForceUpdate();
+
+  const handleCloseInstallSlackAlert = useCallback(() => {
+    setShowSlackInstallAlert(undefined);
+  }, []);
+
+  useEffect(() => {
+    if (queryParams.get('slack_error')) {
+      setShowSlackInstallAlert(queryParams.get('slack_error') as SlackError);
+
+      LocationHelper.update({ slack_error: undefined }, 'replace');
+    }
+  }, []);
+
+  const getRemoveAlertHandler = useCallback((id: string) => {
+    return () => {
+      setItem(id, true);
+
+      forceUpdate();
+    };
+  }, []);
+
+  const store = useStore();
```

Comment:
if some component uses store then it should be placed inside `containers` folder